src/disposition/disposition.controller.ts

```diff
@@ -4,6 +4,7 @@ import {
   Body,
   Req,
   Param,
+  Headers,
```

Comment:
I need to add linting rules for this but if we're not using something it shouldn't import it (I imported it when I was debugging stuff)